examples/bmp280/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"machine"
+	"time"
+	"tinygo.org/x/drivers/bmp280"
+)
+
+func main() {
+	time.Sleep(5 * time.Second)
+
+	machine.I2C0.Configure(machine.I2CConfig{})
+	sensor := bmp280.New(machine.I2C0)
+	sensor.Configure(bmp280.STANDBY_125MS, bmp280.FILTER_4X, bmp280.SAMPLING_16X, bmp280.SAMPLING_16X, bmp280.MODE_FORCED)
+
+	connected := sensor.Connected()
+	if !connected {
+		println("\nBMP280 Sensor not detected\n")
+		return
+	}
+	println("\nBMP280 Sensor detected\n")
+
+	println("Calibration:")
+	sensor.PrintCali()
+
+	for {
+		t, err := sensor.ReadTemperature()
+		if err != nil {
+			println("Error reading temperature")
+		}
+
+		print("Temperature: ")
+		println(t)
+
+		// Temperature in degrees Celsius
+		//fmt.Printf("Temperature: %.2f\n", float32(t)/100)
```

Comment:
I'd remove the commented lines, also, ReadTemperature returns milli Celsiuis, so it should be /1000 instead of /100, or am I missing someting?

Reply:
Thank for catching this one, ReadTemperature() and ReadPressure() now return in milli degrees celsius and milli pascal.

And the example has been adjusted to now properly show the temp in degrees celsius and pressure in hecto pascal.